pkg/storage/cacher.go

```diff
@@ -307,21 +381,53 @@ func (c *Cacher) Codec() runtime.Codec {
 	return c.storage.Codec()
 }
 
+func (c *Cacher) indexValues(event *watchCacheEvent) ([]string, bool) {
+	if c.indexFunc == nil {
+		return nil, false
+	}
+	// TODO: Currently we assume that only one index exists, thus
```

Comment:
Oh, right. Then maybe change this comment so it doesn't claim only one value will be returned :)